bin/qds.py

```diff
@@ -524,6 +542,10 @@ def main():
                          default=False,
                          help="skip verification of server SSL certificate. Insecure: use with caution.")
 
+    optparser.add_option("--cloud", dest="cloud",
+                         default=os.getenv('CLOUD_PROVIDER'),
+                         help="cloud", choices=["AWS", "AZURE", "ORACLE_BMC"])
```

Comment:
I meant this code at line Number 564: 

`     
        if options.api_url is None: 
          options.api_url = "https://api.qubole.com/api/"
`
We should add similar check for cloud.